### DISCUSSION TITLE
Error : "schemeObservable10" is not exported

### DISCUSSION THREAD

**Discussion by sylvainmouquet** [observablehq/plot#2023] [General]
Hello, 
I have an error with latest version : 
`RollupError: node_modules/@observablehq/plot/src/scales/schemes.js (51:2): "schemeObservable10" is not exported by "node_modules/d3/src/index.js", imported by "node_modules/@observablehq/plot/src/scales/schemes.js".`

I am trying to track the error, someone knows the root cause ?
Thanks
Sylvain

**Comment by mbostock** [comment]
You need to update your transitive dependencies. If you want to upgrade everything, you can use `npm update` or `yarn upgrade`. If you just want to fix this error, you’ll need to upgrade `d3-scale-chromatic` which is a transitive dependency of `d3` and `@observablehq/plot`.

**Comment by sylvainmouquet** [comment]
thanks for your help
i use `yarn upgrade-interactive` and everything is up to date 

- d3 -> 7.9.0
- d3-scale-chromatic -> 3.1.0
- @observablehq/plot -> 0.6.14

but the error persists. I'm still trying to find the cause

when i am forcing the version to "@observablehq/plot": "0.6.13" it works

**Reply by mbostock** [reply]
Tough to say. You’ll need to check your yarn.lock file to know for sure. You could also try upgrading everything rather than doing it interactively.

**Reply by sylvainmouquet** [reply]
it's solved. 2 versions of `d3-scale-chromatic` was present in the yarn.lock. I am closing the issue
